packages/app/src/debug/DebugContainer.cy.tsx

```diff
@@ -255,6 +257,51 @@ describe('<DebugContainer />', () => {
       })
     })
 
+    it('does not render DebugPendingRunSplash and DebugNewRelevantRunBar at the same time', () => {
+      cy.mountFragment(DebugSpecsFragmentDoc, {
+        variableTypes: DebugSpecVariableTypes,
+        variables: {
+          hasNextRun: false,
+          runNumber: 1,
+          nextRunNumber: -1,
+        },
+        onResult: (result) => {
+          if (result.currentProject?.cloudProject?.__typename === 'CloudProject') {
+            const test = result.currentProject.cloudProject.runByNumber
+
+            // Testing this to confirm we are "making impossible states impossible" in the UI,
+            // and document the expectation in this scenario. For clarity,
+            // we do not expect a 'RUNNING` current and next run at the same time, so
+            // the data below represents an invalid state.
+
+            result.currentProject.cloudProject.runByNumber = {
```

Comment:
Instead of modifying this on the fly, should we use the fixtures you added such as `FAKE_PROJECT_MULTIPLE_COMPLETED`?

I find the modified on the fly ones kind of hard to read - you need to have a mental modal of exactly what you are modifying in the first place. How about you?